README.md

```diff
@@ -173,22 +173,6 @@ helper function:
 </article>
 ```
 
-#### Validating component _context
-
-```twig
-{% if app.debug %}
-    {#
-       Validate the provided template _context
-       For rules, see: https://github.com/rakit/validation#available-rules
-    #}
-    {{ _context|validate_context({
```

Comment:
Heb je hier nog op gezocht in Github? Ik meen me te herinneren dat dit in sommige projecten wel gebruikt wordt.

Reply:
Klopt, vandaag dat ik het nu pas verwijder in deze nieuwe major versie (had het natuurlijk allang veel eerder eruit willen gooien, maar dat kon om die reden niet)